app/controllers/admin/passwords_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Admin::PasswordsController < Devise::PasswordsController
+  before_action :user_logout_required, only: [:new, :edit]
```

Comment:
Should we limit this to just new and edit? Can we have the callback on all actions on the controller?

Reply:
Can do! I wasn't sure which we wanted. Should it be on all actions in both controllers?

Reply:
I'd prefer all actions, yeah.